src/types/utils.ts

```diff
@@ -26,17 +37,29 @@ import {
     BytesType,
     FixedBytesType,
     FunctionType,
+    IntLiteralType,
     IntType,
     MappingType,
     PackedArrayType,
     PointerType,
+    StringLiteralType,
     StringType,
     TupleType,
     TypeNode,
     UserDefinedType,
     UserDefinition
 } from "./ast";
 
+export const operatorGroups = {
```

Comment:
I guess we should rename this to `binaryOperatorGroups` to clarify that unaries are **not expected** here.

Reply:
Fixed in [51bdea0](https://github.com/ConsenSys/solc-typed-ast/pull/142/commits/51bdea0cbff6131127af6ceb6d1e12c11faaecc5).